CHANGES.md

```diff
@@ -46,7 +46,15 @@ after the corresponding tracker issues.
 
 - For developers: We cleaned up the code of LandmarkGraph. Some of the public
   methods were renamed. This class will undergo further changes in the future.
-  http://issues.fast-downward.org/issue989  
+  <http://issues.fast-downward.org/issue989>
+
+- Debug builds with LP solvers
```

Comment:
People might be interested in _GLIBCXX_DEBUG, so I would suggest referencing it in the "headline". For example, "Debug builds with LP solvers vs. the _GLIBCXX_DEBUG flag".